etcdserver/auth/auth_test.go

```diff
@@ -29,6 +30,16 @@ import (
 
 const testTimeout = time.Millisecond
 
+func init() {
+	GenerateFromPassword = func(password []byte, cost int) ([]byte, error) { return password, nil }
```

Comment:
can you put this to the test function that require this change? and also change it back to what it was after that test?

putting this into init() seems to be dangerous. we might have a test that depends on the default bcrypt behavior.